ansible/roles/ocp4-workload-cluster-autoscale/tasks/workload.yml

```diff
@@ -0,0 +1,47 @@
+# vim: set ft=ansible
+---
+# Implement your Workload deployment tasks here
+- name: get current machinesets
+  shell: "oc get machineset -o name -n openshift-machine-api | awk -F/ '{print $2}'"
```

Comment:
The `|awk` bothered me and i was curious: how to do it without a shell task ? i found those:

Use jsonpath:

```yaml
command:  >-
  oc get machineset -n openshift-machine-api
  -o jsonpath='{range .items[*]}{@.metadata.name}{"\n"}{end}'
```

or go template:

```yaml
command: >-
  oc get machineset -n openshift-machine-api
  -o go-template
  --template='{{range .items}}{{printf "%s\n" .metadata.name}}{{end}}'
```

or use lookup:

```yaml
- name: Fetch a list of namespaces
  set_fact:
    machinesets: "{{ lookup('k8s', api_version='machine.openshift.io/v1beta1', kind='MachineSet') | json_query('[].metadata.name') }}"

# then
- loop: "{{ machinesets }}"
  ...
```

Reply:
So are you saying you will not accept it as awk?